liquidator/index.js

```diff
@@ -90,13 +90,19 @@ async function run(logger, address, pollingDelay, priceFeedConfig, liquidatorCon
 
     // The EMP requires approval to transfer the liquidator's collateral and synthetic tokens in order to liquidate
     // a position. We'll set this once to the max value and top up whenever the bot's allowance drops below MAX_INT / 2.
-    await gasEstimator.update();
-    const collateralToken = await ExpandedERC20.at(await emp.collateralCurrency());
-    const syntheticToken = await ExpandedERC20.at(await emp.tokenCurrency());
-    const currentCollateralAllowance = await collateralToken.allowance(accounts[0], empClient.empAddress);
-    const currentSyntheticAllowance = await syntheticToken.allowance(accounts[0], empClient.empAddress);
+    const [collateralToken, syntheticToken] = await Promise.all([
```

Comment:
Should we batch these with the above Promise.all?

Reply:
ye, can do & done.